### DISCUSSION TITLE
Auto complete on recipe search

### DISCUSSION THREAD

**Discussion by monksy** [mealie-recipes/mealie#2825] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

It's hard to tell what you're searching for may not actually have something there. 

### Please provide a concise description of the feature that would resolve your issue.

Add in autocomplete on search.

### Please consider and list out some caveats or tradeoffs made in your design decision

Puts more demand on the backend. 

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [X] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by boc-the-git** [comment]
The meal planner already does it.

I'm assuming you mean on the home page? 

Might be an easy retrofit into there (haven't looked).

**Comment by RealFoxie** [comment]
The search is fuzzy, so autocompleting might be weird?
What is the actual problem to fix? Is it a feedback problem where it doesn't show when it finished loading the search request?

**Comment by monksy** [comment]
Closing because it looks like it does this.. I didn't notice this before may have been on a much older version when I wrote the feature request down.
